DevVehicle35-Motors/App/Program.cs

```diff
@@ -11,6 +11,7 @@
     Console.WriteLine("Select the type of vehicle you want to build:");
     Console.WriteLine("2. Bus");
     Console.WriteLine("3. Quad");
+    Console.WriteLine("4. Van");
```

Comment:
Number 4 is already used and I think 5 and 6 are too

Reply:
Thanks for the advice! We moved our Feature to option 7